### PR TITLE
PLT-2076: Add help text below the input box when a user starts typing

### PR DESCRIPTION
**Pull request by rodrigocorsi2**

### CODE REVIEW

**Comment by it33** [comment]
Sweet!! Thanks!! Setting up test server for team to have a look

File: web/react/components/textbox.jsx [hunk 1]
```diff
@@ -129,12 +129,12 @@ export default class Textbox extends React.Component {
     }
 
     render() {
+        const hastext = this.props.messageText.length > 0;
```
Comment (jwilander) [inline web/react/components/textbox.jsx]: Very minor, but could you use camel case for this variable name(aka `hasText` over `hastext`)? It's mentioned in the [style guide here](http://docs.mattermost.com/developer/style-guide.html#variables)

**Comment by jwilander** [comment]
+1

**Comment by jwilander** [comment]
@hmhealey please take a look

**Comment by hmhealey** [comment]
+1 Looks good

**Pull request merged**